libs/output-mapping/src/Writer/Helper/SliceHelper.php

```diff
@@ -64,14 +64,11 @@ private function validateMappingSource(MappingSource $source): void
                 && $mapping['delimiter'] !== Manifest::DEFAULT_DELIMITER;
             $hasNonDefaultEnclosure = isset($mapping['enclosure'])
                 && $mapping['enclosure'] !== Manifest::DEFAULT_ENCLOSURE;
-            if ($hasNonDefaultDelimiter || $hasNonDefaultEnclosure) {
-                throw new SliceSkippedException(
-                    'Params "delimiter" or "enclosure" specified in mapping are not supported by slicer.',
-                );
-            }
-            if (isset($mapping['columns']) && $mapping['columns'] !== []) {
-                throw new SliceSkippedException(
-                    'Param "columns" specified in mapping is not supported by slicer.',
+            $hasColumns = isset($mapping['columns']) && $mapping['columns'] !== [];
+
+            if ($hasNonDefaultDelimiter || $hasNonDefaultEnclosure || $hasColumns) {
+                throw new InvalidOutputException(
```

Comment:
Nechal jsem to tu, abych to nemusel cely prekopavat a stejne se to vyhledove vyhodi pryc.